### DISCUSSION TITLE
max的access插件为什么不在app.ts里配置?

### DISCUSSION THREAD

**Discussion by PopperLi** [umijs/umi#11586] [General]
大多数插件的运行时都在app.ts里配置，为什么access需要单独写一个文件？

**Comment by fz6m** [comment]
从 umi 3 开始 access 就是这么约定的，延续历史所以是单独的一个文件。

**Reply by PopperLi** [reply]
有没有考虑在app.ts中也支持配置呢？毕竟都属于运行时的能力
